HALO/wales/main.yaml

```diff
@@ -0,0 +1,35 @@
+sources:
+  cloudparameter:
+    args:
+      path: "{{CATALOG_DIR}}/wales_cloudtop.yaml"
+    description: 'WALES Lidar cloud top heights'
+    driver: intake.catalog.local.YAMLFileCatalog
+    metadata: {}
+
+  wv:
+    args:
+      path: "{{CATALOG_DIR}}/wales_wv.yaml"
+    description: 'WALES Lidar Water Vapour Mixing Ratio in g/kg'
+    driver: intake.catalog.local.YAMLFileCatalog
+    metadata: {}
+
+  adepg:
+    args:
+      path: "{{CATALOG_DIR}}/wales_adepg.yaml"
+    description: 'WALES Lidar Particle Depolarisation at 532 nm'
```

Comment:
The dataset (HALO-0205) says "aerosol depolarisation at 532 nm". I am not entirely sure if the distinction between "aerosol" and "Particle" makes any difference (probably not), but still I am wondering why you chose "Particle".

Reply:
In the corresponding quicklook plots the WALES people wrote "particle" that's way I chose that term. However "aerosol" better fits their own abbreviation "adepg" - **aerosol** depolarisation green. I change it to aerosol :)